contracts/FlowFees.cdc

```diff
@@ -70,6 +71,59 @@ pub contract FlowFees {
         }
     }
 
+    // VerifyPayerBalanceResult is returned by the verifyPayersBalanceForTransactionExecution function
+    pub struct VerifyPayerBalanceResult {
+        // True if the payer has sufficient balance for the transaction execution to continue
+        pub let canExecuteTransaction: Bool
+        // The minimum payer balance required for the transaction execution to continue
+        pub let requiredBalance: UFix64
+        // The maximum fees the transaction can incur (if all available execution effort is used)
+        pub let maximumTransactionFees: UFix64
```

Comment:
Is this the `requiredBalance + the max execution effort` or is it just the max execution effort?